### ISSUE TITLE
Castle Oztroja Coffer

### ISSUE THREAD

**Issue by Swankyjr** [EdenServer/community#1247]
So i was in Castle O last night with a party and they where helping me get the Coffer key, for my DRK AF gear and after we got the key we did not see the chest for about 2 hours.  We where the only ones past the Brass door where the coffers start spawning. Now i'm not 100% sure if someone was taking it or not, but as far as i know (and asked) no one was taking it while i was looking for the chest.

**Comment by Mikokiko** [comment]
It been on lockdown by Ecks / Kalibur. He never get mimic and pop it every 30 min as a lvl 50 THF. Getting the full gil on each char. 

He has been there yesterday and today. Good luck trying to get your AF. Since w/o knowing the pop time it's just a waiste of time.

**Comment by Redenius** [comment]
If a THF used tools to open a coffer, it will extend the time. I know I was in Castle Oztroja a few days ago for AF  . . . and I was on THF :)

**Comment by jaheny13** [comment]
Not sure if adjustments were made after this post, but for the last week or so picking coffers as a THF has been a bit of a nightmare. Two days ago, it took me 33 Living Keys to open a single coffer that wasn't a mimic. 

In the past, I would break 0-3 keys per coffer, but I'm always breaking at least 5-8 keys per coffer now. Friend has been having a similar experience in Fei'Yin.

**Comment by godmode** [comment]
What level is your THF?

**Comment by jaheny13** [comment]
My THF is level 55. My friend's is 52.

**Comment by godmode** [comment]
Sounds like it’s about right for that level at 3-8. 33 seems fairly excessive though. Is that very rarely hit?

**Comment by jaheny13** [comment]
I honestly don't know if that was just a streak of bad luck or not. I only stayed out of stubbornness and left after I finally got it to open. I haven't gone back since I was losing money being there. 

I had gone and picked coffers a few times in my spare time after I hit 50 in both Fei'Yin and Castle O for a week or two before this post. I had decent enough success, but I'm breaking at least 5 keys now on every coffer and most are more than that. 

Not trying to make this sound dramatic. We both could just be going through some really bad luck streaks, but figured I'd mention it in case any adjustments were made.
